pkg/controllers/internalmembercluster/v1beta1/member_suite_test.go

```diff
@@ -402,7 +402,7 @@ var _ = BeforeSuite(func() {
 
 	// This controller is created for testing purposes only; no reconciliation loop is actually
 	// run.
-	workApplier2 = workapplier.NewReconciler(hubClient, member2ReservedNSName, nil, nil, nil, nil, 0, 1, time.Second*5, time.Second*5, time.Minute, true, 60)
+	workApplier2 = workapplier.NewReconciler(hubClient, member2ReservedNSName, nil, nil, nil, nil, 0, 1, time.Minute, true, 60, nil)
```

Comment:
Why setting the 	watchWorkWithPriorityQueue as true?

Reply:
Ah, it has been this way in the current master branch; I have patched it out.